lib/pool.py

```diff
@@ -97,6 +99,7 @@ def first_shared_sr(self):
         return None
 
     def save_uefi_certs(self):
+        assert self.master.xcp_version < version.parse("8.3"), "function only relevant on XCP-ng 8.2"
```

Comment:
Messages showing the reasons would be more useful.  Maybe "8.3+ only saves custom certificates, not all of them" here?